packages/ember/package.json

```diff
@@ -31,10 +31,12 @@
     "pack": "npm pack"
   },
   "dependencies": {
+    "@embroider/macros": ">= 0.25.0",
     "@sentry/browser": "5.24.2",
     "@sentry/tracing": "5.24.2",
     "@sentry/types": "5.24.2",
     "@sentry/utils": "5.24.2",
+    "@types/babel__core": "^7.1.9",
```

Comment:
Why did you add `babel__core`, which type was missing?

Reply:
Hmm it resulted to type errors without it, but I just checked again, not anymore! But I have updated @embroider/macros a few versions in between, that probably improved somewhat there. I removed it!